lamindb_setup/_init_instance.py

```diff
@@ -95,11 +95,16 @@ def register_user(usettings):
         pass
 
 
-def register_user_and_storage_in_instance(isettings: InstanceSettings, usettings):
-    """Register user & storage in DB."""
+def register_initial_objects(isettings: InstanceSettings, usettings):
+    """Register space, user & storage in DB."""
     from django.db.utils import OperationalError
+    from lamindb.models import Space
 
     try:
+        Space.objects.get_or_create(
```

Comment:
This is bad because it'll assign a random uid. The other logic we have assigns a uid that indicates that the "All" space is special.

Reply:
it was always so, hm, nothing new here.

Reply:
Hmmm, I see you're just moving it.

Reply:
When was this introduced? I remember something that would populate `uid="000000000000"` (12 0s).

Reply:
i can add this.

Reply:
but it won't have a random uid, it will have exactly `"00000000"`. See https://github.com/laminlabs/lamindb/blob/01abda450218a9647b1f23b5368193425a0fcf7a/lamindb/models/record.py#L896

Reply:
It has been always like this, no changes. Also note that there is no auto-increment on space `uid`, so to create anew space `uid` has to be manually provided like i do here https://github.com/laminlabs/lamindb/blob/01abda450218a9647b1f23b5368193425a0fcf7a/tests/permissions/scripts/setup_access.py#L5

Reply:
All good then!

Reply:
Perfect.

Reply:
> Also note that there is no auto-increment on space uid

It shouldn't, it should be a random uid created in central

Reply:
> so to create anew space uid has to be manually provided like i do here https://github.com/laminlabs/lamindb/blob/01abda450218a9647b1f23b5368193425a0fcf7a/tests/permissions/scripts/setup_access.py#L5

This is not how it should be used but for testing it's OK. This should be a random uid.

Reply:
Ok, got it.